distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1305,7 +1306,7 @@ type Table
                 new_ctx = Context.for_subquery new_from . set_where_filters where_expressions
                 Table.Value new_table_name self.connection result_columns new_ctx
 
-    ## ALIAS cartesian join
+    ## ALIAS cartesian join append
```

Comment:
```suggestion
    ## ALIAS cartesian join, append
```

I think we were usually delimiting with a `,`.

Moreover, I think this needs to be replicated to in-memory table as well, right?

Reply:
Yes. Sorry, thanks for catching. Fixed.